src/node-red/hardware/opto-22-groove-rio-7-mm2001-10.md

```diff
@@ -0,0 +1,57 @@
+---
+eleventyNavigation:
+  key: GRV-R7-MM2001-10
+  parent: Hardware
+meta:
+  title: Setting up Node-RED on Opto-22 Groov Rio R7
+  description: Learn how to install and configure Node-RED on the Opto-22 Groov Rio R7, a rugged edge I/O module for industrial applications.
+  keywords: node-red, flowfuse, Opto-22 Groov Rio, groov RIO, industrial IoT, edge computing
+image: "/node-red/hardware/images/GRV-R7-MM1001-10.jpeg"
+specifications:
+  Model: GRV-R7-MM2001-10
+  RAM: 1 GB DDR3
+  Processor: ARM Cortex-A8, 1 GHz
+  I/O Channels: 10 multi-signal, multifunction channels (analog I/O, temperature, discrete I/O, mechanical relays)
+  Connectivity:
+    - Dual switched Gigabit Ethernet
+    - USB 2.0 (host)
+    - Power over Ethernet (PoE)
+    - 10–32 VDC power input
+  Clock Speed: 1 GHz
+  Storage:
+    - 4 GB eMMC (internal)
+    - USB memory stick support (up to 32 GB)
+layout: layouts/hardware.njk
+---
+
+The Opto-22 Groov Rio R7 is a rugged edge I/O module designed for industrial applications. Equipped with a powerful ARM Cortex-A8 processor, versatile I/O channels, and various connectivity options, it’s an ideal solution for edge computing and industrial IoT.
+
+## Prerequisites
+
+Before proceeding with the installation, ensure you have the following:
+
+- **Opto-22 Groov Rio R7** – A functioning device with internet access.
+- **FlowFuse Account** - Ensure you have a FlowFuse account. If not, you can create a free account that allows you to manage up to two edge devices for free. For more information, refer to [FlowFuse Free Tier](/blog/2024/12/flowfuse-release-2-12/)
```

Comment:
Why not redirect to create an account? Why sell the free tier? This is a professional device